.forge-snapshots/swap with fee and rebalance.snap

```diff
@@ -0,0 +1 @@
+187820
```

Comment:
hold up, this is only 1872 more gas than `swap with fee`. Does this mean rebalancing only costs 1872 rn??? If so that's great......... But I feel like too good to be true

Reply:
no, the rebalancing is happening after the swap in this test (its happening before a modifyPosition). i can change the name of this snap haha